File: trash/flores_gomez_javier_alberto/usuario.py
from libro import Libro


class Usuario:
    def __init__(self, nombre_usuario: str, password: str, codigo: int):
        self._nombre_usuario = nombre_usuario
        self._password = password
        self._codigo_usuario = codigo
        self._libros_prestados = []
        self._libros_reservados = Libro
        self._reserva = False

    @property
    def nombre_usuario(self):
        return self._nombre_usuario

    @property
    def password(self):
        return self._password

    @property
    def codigo_usuario(self):
        return self._codigo_usuario

    @property
    def libros_prestados(self):
        return self._libros_prestados

    @property
    def libros_reservados(self):
        return self._libros_reservados

    @property
    def reserva(self):
        return self._reserva

    @nombre_usuario.setter
    def nombre_usuario(self, nombre):
        self._nombre_usuario = nombre

    @password.setter
    def password(self, password):
        self._password = password

    @codigo_usuario.setter
    def codigo_usuario(self, codigo):
        self._codigo_usuario = codigo

    @libros_reservados.setter
    def libros_reservados(self, libro):
        self._libros_reservados = libro

    @reserva.setter
    def reserva(self, reserva):
        self._reserva = reserva

File: development_(codes)/src/edd_proyecto/Usuario.java
package edd_proyecto;

import java.util.ArrayList;
import java.util.List;

public class Usuario {
    private String nombreUsuario;
    private String password;
    private int codigoUsuario;
    private int tipoUsuario;
  

    public Usuario(String nombreUsuario, String password, int codigoUsuario, int tipoUsuario) {
        this.nombreUsuario = nombreUsuario;
        this.password = password;
        this.codigoUsuario = codigoUsuario;
        this.tipoUsuario=tipoUsuario;
      
    }

    public String getNombreUsuario() {
        return nombreUsuario;
    }

    public String getPassword() {
        return password;
    }

    public int getCodigoUsuario() {
        return codigoUsuario;
    }


    public void setNombreUsuario(String nombre) {
        this.nombreUsuario = nombre;
    }

    public void setPassword(String password) {
        this.password = password;
    }

    public void setCodigoUsuario(int codigo) {
        this.codigoUsuario = codigo;
    }

	public int getTipoUsuario() {
		return tipoUsuario;
	}

	public void setTipoUsuario(int tipoUsuario) {
		this.tipoUsuario = tipoUsuario;
	}

   
}





